src/protocol/graphics.ts

```diff
@@ -255,11 +264,8 @@ export function setupGraphics(store: UIStore) {
           }
         }
         paintGraphics(screenShot, mouse);
-      } else {
-        const { screen, mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
-        paintGraphics(screen, mouse);
       }
-    }
+    } catch {}
```

Comment:
What error are we expecting here? If it's just a general purpose catch-all, can we still call Sentry even if we stop the error from propogating?

Reply:
We're expecting a [DownloadCancelledError](https://github.com/RecordReplay/devtools/blob/73b26477108a666885b40150c3ae7b62fb80e5b8/src/protocol/screenshot-cache.ts#L6) because screenshot downloads are cancellable (they are cancelled when too many screenshot downloads are requested at the same time, although this was more relevant in the past when the timeline tooltip showed a thumbnail of the screenshot from the hovered time - that could queue a lot of screenshot downloads, but it would only ever use the last one, so it made sense to cancel the others).
I'll check the error and rethrow if it's not a `DownloadCancelledError`.